docs/src/starknet/script.md

```diff
@@ -179,10 +181,12 @@ $ sncast \
   --account example_user \
   script map_script
 
-[DEBUG]	Contract address               	(raw: 0x436f6e74726163742061646472657373
-[DEBUG]	                               	(raw: 0x6f9492c9c2751ba5ccab5b7611068a6347d7b313c6f073d2edea864f062d730
-[DEBUG]	Invoke tx hash                 	(raw: 0x496e766f6b652074782068617368
-[DEBUG]	                               	(raw: 0x60175b3fca296fedc38f1a0ca30337ae666bb996a43beb2f6fe3a3fa90d3e6b
+Class hash of the declared contract: 685896493695476540388232336434993540241192267040651919145140488413686992233
```

Comment:
I think at the moment we cannot display it as hex, right? We should create an issue to add such feature, if possible

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1668